server/controllers/medical/snis.js

```diff
@@ -290,11 +290,22 @@ function getReport (req, res) {
 	}
 }
 
+function get_snis_zs (req, res) {
```

Comment:
A better name for this would be `function healthZones()`.

Why are you not using `next()`?!?